globals_test.go

```diff
@@ -1067,21 +1068,28 @@ func TestLoadGlobals(t *testing.T) {
 
 			wantGlobals := tcase.want
 
-			stacks := s.LoadStacks()
-			for _, stack := range stacks {
-				stackMeta := stack.Meta()
-				got, err := terramate.LoadStackGlobals(s.RootDir(), stackMeta)
+			stackEntries, err := terramate.ListStacks(s.RootDir())
+			if errors.IsKind(tcase.wantErr, hcl.ErrTerramateSchema) {
+				errors.AssertKind(t, err, tcase.wantErr)
+			}
```

Comment:
When tcase.wantErr != hcl.ErrTerramateSchema (like nil) is it safe/best to go on with the test if listing stacks failed ? I imagine the stackEntries will be an empty slice and the test failure will be harder to understand because there will be no indication that stack listing failed (the error will indicate that something wrong happened loading the globals) :thinking:

Reply:
fixed